### PR TITLE
Allow connecting to multiple brokers 

### PR DESCRIPTION
**Pull request by Revolyssup**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — APPROVED** [review]

File: internal/graphql/model/operator_helper.go [hunk 1]
```diff
@@ -172,6 +176,7 @@ func SubscribeToBroker(provider models.Provider, mesheryKubeClient *mesherykube.
 		if err == nil && broker.Status.Endpoint.External != "" {
 			break
```
Comment (leecalcote) [inline internal/graphql/model/operator_helper.go:177]: @Revolyssup the error handling here seems like it needs improvement.

File: internal/graphql/model/helpers.go [hunk 2]
```diff
@@ -291,3 +291,31 @@ func SetOverrideValues(delete bool, adapterTracker models.AdaptersTrackerInterfa
...
+}
+
+func (k *K8sConnectionTracker) Get(id string) (url string) {
+	k.mx.Lock()
+	defer k.mx.Unlock()
+	url = k.ContextToBroker[id]
+	return
+}
+
+func (k *K8sConnectionTracker) Log(l logger.Handler) {
```
Comment (leecalcote) [inline internal/graphql/model/helpers.go:313]: A couple of comments on these functions would be helpful.

File: internal/graphql/model/operator_helper.go [hunk 3]
```diff
@@ -206,20 +211,36 @@ func SubscribeToBroker(provider models.Provider, mesheryKubeClient *mesherykube.
...
 		MaxReconnect:   5,
 	})
 	// Hack for minikube based clusters
-	if err != nil {
+	if err != nil && conn == nil {
 		return endpoint, err
 	}
+	defer func() {
+		if conn == nil {
+			return
```
Comment (leecalcote) [inline internal/graphql/model/operator_helper.go:230]: Is connection state being tracked anywhere? If so, should this update that state?

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: internal/graphql/model/operator_helper.go (hunk 3 above)
Comment (Revolyssup) [reply internal/graphql/model/operator_helper.go:230]: This connection is local to the function call SubscribeToBroker so it doesn't need to be tracked anywhere else. 
Just few lines below, you will find a DeepCopy method being invoked which actually copies this local instance of connection into globally maintained broker connection if no errors are encountered.

File: internal/graphql/model/helpers.go (hunk 2 above)
Comment (Revolyssup) [reply internal/graphql/model/helpers.go:313]: Sure

File: internal/graphql/model/operator_helper.go (hunk 1 above)
Comment (Revolyssup) [reply internal/graphql/model/operator_helper.go:177]: I think it is fine. It spins in a loop with a timeout until we can get the broker without any errors AND it has been assigned an External IP. This logic seems fine to me. Can you point out what part of it needs improvement?

**Review by leecalcote — APPROVED** [review]

**Pull request merged**